src/App.jsx

```diff
@@ -1,10 +1,22 @@
 import { ChakraProvider, Button, Image } from '@chakra-ui/react';
 import React, { useState } from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module 'react-router-dom'.